package.json

```diff
@@ -68,7 +70,7 @@
         "react-dom": "^17.0.1",
         "ts-jest": "^24.3.0",
         "ts-loader": "^8.0.18",
-        "typescript": "^4.1.3",
+        "typescript": "4.1.3",
```

Comment:
Why pin to this version? We recommend keeping the caret: https://github.com/guardian/recommendations/blob/main/dependencies.md#javascript-npm-dependencies

Reply:
It's in the PR description, having the caret causes failures